core/domain/story_domain.py

```diff
@@ -231,66 +273,32 @@ def to_dict(self):
             'language_code': self.language_code,
             'schema_version': self.schema_version,
             'version': self.version,
-            'story_nodes': [
-                node.to_dict() for node in self.story_nodes
-            ]
+            'story_contents': self.story_contents.to_dict()
         }
 
     @classmethod
-    def from_dict(
-            cls, story_dict,
-            story_created_on=None, story_last_updated=None):
-        """Return a Story domain object from a dict.
-
-        Args:
-            story_dict: dict. The dictionary representation of the
-                story.
-            story_created_on: datetime.datetime. Date and time when the
-                story is created.
-            story_last_updated: datetime.datetime. Date and time when
-                the story was last updated.
-
-        Returns:
-            Story. The corresponding Story domain object.
-        """
-        story = cls(
-            story_dict['id'], story_dict['title'],
-            story_dict['description'], story_dict['notes'],
-            [
-                StoryNode.from_dict(node_dict)
-                for node_dict in story_dict['story_nodes']
-            ], story_dict['schema_version'], story_dict['language_code'],
-            story_dict['version'], story_created_on, story_last_updated)
-
-        return story
-
-    @classmethod
-    def create_default_story(
-            cls, story_id, title=feconf.DEFAULT_STORY_TITLE,
-            description=feconf.DEFAULT_STORY_DESCRIPTION,
-            notes=feconf.DEFAULT_STORY_NOTES,
-            language_code=constants.DEFAULT_LANGUAGE_CODE):
-        """Returns a story domain object with default values.
+    def create_default_story(cls, story_id):
+        """Returns a story domain object with default values. This is for the
+        the frontend where a default blank story would be shown to the user
```

Comment:
"the" is duplicated (see end of last line)

Reply:
Done.